v2/passwordless/serverless/with-aws-lambda/jwt-authorizer.mdx

```diff
@@ -75,10 +86,76 @@ function getBackendConfig(): SuperTokensTypes.TypeInput {
 
 module.exports.getBackendConfig = getBackendConfig;
 
+```
+</TabItem>
+
+<TabItem value="python">
+
+```python title="config.py"
+from supertokens_python.recipe import emailpassword, session
+from supertokens_python import (
+    InputAppInfo,
+    SupertokensConfig,
+)
+
+supertokens_config = SupertokensConfig(
+    ^{coreInjector_connection_uri_comment_with_hash}
+    connection_uri=^{coreInjector_uri}
+    ^{coreInjector_api_key_commented_with_hash}api_key=^{coreInjector_api_key}
+)
+
+app_info = InputAppInfo(
+    # learn more about this on https://supertokens.com/docs/emailpassword/appinfo
+    app_name="^{form_appName}",
+    api_domain="^{form_apiDomain}",
+    website_domain="^{form_websiteDomain}",
+    api_base_path="^{form_apiBasePath}",
+    website_base_path="^{form_websiteBasePath}",
+    api_gateway_path="/dev",
```

Comment:
make similar changes to the node code snippet as well in this section to use values from the appInfo component

Reply:
Done.